app/assets/javascripts/index/history-changesets-layer.js

```diff
@@ -1,15 +1,40 @@
 OSM.HistoryChangesetsLayer = L.FeatureGroup.extend({
-  _changesets: [],
+  newerColor: "#AA8866",
+  olderColor: "#8888AA",
```

Comment:
If we're going to use named constants for these colours shouldn't we do it for the others as well?

Reply:
Includes workaround for https://github.com/leaflet/leaflet/issues/2662